
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">

    <title>Computational Biology Research Team</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300|Sonsie+One" rel="stylesheet" type="text/css">
    <link rel="stylesheet" href="/assets/css/spectre.css">
    <link rel="stylesheet" href="/assets/css/index.css">

  </head>

  <body>
    <div id="particles-js"></div>
    <nav class="navbar pa1">
      <div class="container">
        <div class="columns flex-between-md">
        <section class="column col-md-12 col-lg-6 flex flex-between-md">
        <div class="flex-column">
          <div class="h6">Computer Science Department @ ISU</div>
          <div class="h1">Computational Biology Research Team</div>
        </div>
        </section>
        <section class="column col-md-12 col-lg-6 flex flex-end flex-between-md pa1">
            
                <a href="/" class="btn btn-link mt1 h5">Home</a>
            
                <a href="/team" class="btn btn-link mt1 h5">Our Team</a>
            
                <a href="/projects" class="btn btn-link mt1 h5">Projects</a>
            
                <a href="/publications" class="btn btn-link mt1 h5">Publications</a>
            
        </section>
        </div>
      </div>
    </nav>

    <main class="header-margin">
        <div class="container">
        <div class="columns flex-center">

          <div class="column col-6 col-md-12 pa2 bg-blur full-shadow rounded-corner">

              <h2>Projects</h2>
<div class="card">
<div class="card-image float-left">
        <img src="/assets/images/projects/phylo-rs.png" class="img-w-responsive">
</div>
<div class="card-content">
    <div class="card-header">
    <div class="card-title h3">Phylo-rs</div>
    </div>
    <div class="card-body">
        Phylo is a fast, extensible, general-purpose, and WebAssembly-capable library for phylogenetic analysis and inference written in Rust. Phylo-rs leverages a combination of memory safety, speed, and native WebAssembly support offered by Rust to provide a robust set of memory-efficient data structures with basic algorithms ranging from tree manipulation with SPR to computing tree statistics such as phylogenetic diversity.
    </div>
    <div class="card-footer">
    <a href="https://github.com/sriram98v/phylo-rs" class="btn btn-primary">GitHub</a>
    </div>
</div>
</div>
<div class="card">
<div class="card-image float-left">
        <img src="." class="img-w-responsive">
</div>
<div class="card-content">
    <div class="card-header">
    <div class="card-title h3">ml-seqclass</div>
    </div>
    <div class="card-body">
        Ml-seqclass is a tool that aligns reads to references by first building a suffix array, then aligning the reads to the references by finding seed matches and extending them. After finding all alignments, an EM algorithm is employed to identify the most-likely true source for each read, and to estimate population proportions of the detected references.
    </div>
    <div class="card-footer">
    <a href="https://github.com/sriram98v/ml-seqclass" class="btn btn-primary">GitHub</a>
    </div>
</div>
</div>
<div class="card">
<div class="card-image float-left">
        <img src="/assets/images/projects/MOG.png" class="img-w-responsive">
</div>
<div class="card-content">
    <div class="card-header">
    <div class="card-title h3">MetaOmGraph</div>
    </div>
    <div class="card-body">
        MetaOmGraph (MOG) is a Java software to interactively explore and visualize large datasets. MOG is user-centered software, designed for all types of data, but specialized for expression data. It combines the ability to analyze very large data sets in real time with metadata analysis, statistical analysis, list-making, and graphing capabilities.
    </div>
    <div class="card-footer">
    <a href="https://github.com/eswlab/MetaOmGraph" class="btn btn-primary">GitHub</a>
    </div>
</div>
</div>
<div class="card">
<div class="card-image float-left">
        <img src="/assets/images/projects/pd_stats.jpeg" class="img-w-responsive">
</div>
<div class="card-content">
    <div class="card-header">
    <div class="card-title h3">PD Stats</div>
    </div>
    <div class="card-body">
        PD Stats is a project designed with algorithms to compute Phylogenetic Diversity (PD) Statistics - Minimum, Maximum, Average, Variance on a phylogenetic tree.
    </div>
    <div class="card-footer">
    <a href="https://github.com/flu-crew/PD_stats" class="btn btn-primary">GitHub</a>
    </div>
</div>
</div>
<div class="card">
<div class="card-image float-left">
        <img src="/assets/images/projects/iGTP.png" class="img-w-responsive">
</div>
<div class="card-content">
    <div class="card-header">
    <div class="card-title h3">iGTP</div>
    </div>
    <div class="card-body">
        iGTP is a software package for phylogenetic analyses using gene tree parsimony (GTP).
    </div>
    <div class="card-footer">
    <a href="https://github.com/CompBioTeam/iGTP" class="btn btn-primary">GitHub</a>
    </div>
</div>
</div>
<div class="card">
<div class="card-image float-left">
        <img src="." class="img-w-responsive">
</div>
<div class="card-content">
    <div class="card-header">
    <div class="card-title h3">DupTree</div>
    </div>
    <div class="card-body">
        DupTree is a program for phylogenetic analyses using gene tree parsimony. That is, given a collection of binary gene trees, DupTree searches for a species supertree that implies the fewest number of gene duplication events.
    </div>
    <div class="card-footer">
    <a href="https://github.com/CompBioTeam/DupTree" class="btn btn-primary">GitHub</a>
    </div>
</div>
</div>
<div class="card">
<div class="card-image float-left">
        <img src="." class="img-w-responsive">
</div>
<div class="card-content">
    <div class="card-header">
    <div class="card-title h3">RF-Supertrees</div>
    </div>
    <div class="card-body">
        This program takes as input a collection of rooted trees and heuristically searches for a binary supertree that minimizes the total (rooted) Robinson-Foulds distance between the supertree and the input trees.
    </div>
    <div class="card-footer">
    <a href="https://github.com/CompBioTeam/RF-Supertrees" class="btn btn-primary">GitHub</a>
    </div>
</div>
</div>
<div class="card">
<div class="card-image float-left">
        <img src="." class="img-w-responsive">
</div>
<div class="card-content">
    <div class="card-header">
    <div class="card-title h3">biclique</div>
    </div>
    <div class="card-body">
        This is an implementation of maximal biclique enumeration algorithm from Alexe et al. (http://citeseer.nj.nec.com/alexe02consensus.html)
    </div>
    <div class="card-footer">
    <a href="https://github.com/CompBioTeam/biclique" class="btn btn-primary">GitHub</a>
    </div>
</div>
</div>


          </div>
        <div class="panel">
<div class="panel-header">
  <div class="panel-title text-center text-bold text-large">News</div>
</div>
<div class="panel-body">
    <div class="tile">
        <div class="tile-content">
            <a href="#" class="tile-title">Sanket Wagle presents "The Asymetric Cluster Affinity Cost" at Recomb-CG.</a>
        </div>
      </div>
      <div class="tile">
        <div class="tile-content">
            <a href="#" class="tile-title">Phylogenetic Diversity Statistics for all Clades in a Phylogeny, S. Grover, A. Markin,.T. Anderson, O. Eulenstein accepted by ISMB 2023.</a>
        </div>
      </div>
      <div class="tile">
        <div class="tile-content">
            <a href="#" class="tile-title">The Asymetric Cluster Affinity Cost,S. Wagle, A. Markin, P. Gorecki, T. Anderson, O. Eulenstein accepted by Recomb-CG 2023.</a>
        </div>
      </div>
</div>
        

        </div>
        </div>
        </div>

    </main>
        <footer class="flex flex-right">
      <p class="h6">Website designed by <a href="https://sriram98v.github.io/">Sriram Vijendran</a> Â©Copyright 2050 by nobody. All rights reversed.</p>      
    </footer>


  <script src="/assets/js/particles.js" ></script>
  <script src="/assets/js/app.js" ></script>
  </body>
</html>